documentation/user_docs/docs/manual/Manipulating_and_extracting_data_from_SQW_files_and_objects.rst

```diff
@@ -2,166 +2,367 @@
+This is a function to give the header information in an SQW file or file to which an sqw object has been saved, whose
+full filename is given by the argument ``filename``. If the option ``'-full'`` is used then a fuller set of header
+information, rather than just the principal header, is returned.
 
 
 head_dnd
 ========
 
-::
+.. code-block:: matlab
 
-   info=head_dnd(filename);
+   info = head_dnd(filename);
 
 
-This is a function to give the header information in file to which a dnd object has been saved, whose full filename is given by the argument ``filename``.
+This is a function to give the header information in file to which a dnd object has been saved, whose full filename is
+given by the argument ``filename``.
 
 read_horace
 ===========
 
-::
+.. code-block:: matlab
 
-   output=read_horace(filename);
+   output = read_horace(filename);
 
 
-This is a function to read sqw or dnd data from a file. The object type is determined from the contents of the file. If the file contains a full sqw dataset (whether created using gen_sqw or as the result of saving a cut), the returned variable is an sqw object; if the file contains a dnd dataset, the output is the corresponding d01, d1d, ...or d4d object.
+This is a function to read sqw or dnd data from a file. The object type is determined from the contents of the file. If
+the file contains a full sqw dataset (whether created using gen_sqw or as the result of saving a cut), the returned
+variable is an sqw object; if the file contains a dnd dataset, the output is the corresponding d01, d1d, ...or d4d
+object.
 
 read_sqw
 ========
 
-::
+.. code-block:: matlab
 
-   output=read_sqw(filename);
+   output = read_sqw(filename);
 
-This is a function to read sqw data from a file. Note that in this context we mean an n-dimensional dataset, which includes pixel information, that has been saved to file. This could be either a full SQW file created wusing gen_sqw, or an sqw dataset that has been saved to file. The object ``output`` will be an sqw object.
+This is a function to read sqw data from a file. Note that in this context we mean an n-dimensional dataset, which
+includes pixel information, that has been saved to file. This could be either a full SQW file created wusing gen_sqw, or
+an sqw dataset that has been saved to file. The object ``output`` will be an sqw object.
 
 
 read_dnd
 ========
 
-::
+.. code-block:: matlab
 
-   output=read_dnd(filename);
+   output = read_dnd(filename);
 
 
-Exactly the same as above, but reads dnd data saved to file. If the file contains full sqw dataset, then it will be read as if it contained just a dnd dataset.
+Exactly the same as above, but reads dnd data saved to file. If the file contains full sqw dataset, then it will be read
+as if it contained just a dnd dataset.
```

Comment:
there was a time when a sqw (or dnd) could be constructed from the filename e.g. sqw(filename). Is this now deprecated?

Reply:
No, that's a more recent addition. This is old stuff